src/components/header/header.module.css

```diff
@@ -0,0 +1,122 @@
+.logo{
+    width: 110px;
+    height: 70px;
+}
+.navbarContainer{
+    background: #FFFFFF;
+    box-shadow: 0px 4px 4px rgba(0, 0, 0, 0.25);
+    padding: 0;
+}
+.menuButton{
+  border: none;
+  background-color: #FFFFFF;
+}
+.toggleIcon{
+  font-size: 15px;
+  border: 1px solid #000000;
+  padding: 0.5em;
+  border-radius: 10px;
+}
+.navigationMenu{
+  display: flex;
+  justify-content: flex-end !important;
+}
+.menuItems{
+  margin-right: 1rem;
+  margin-left: 1rem;
+  text-decoration: none;
+  color: #000000;
+  font-size: 16px;
+  font-weight: 400;
+}
+.menuItems:nth-child(1){
+  font-weight: 700;
+}
+.linksButtons{
+  display: flex;
+  align-items: center;
+  justify-self: center;
+  text-decoration: none;
+}
+.logInButton{
+  padding: 10px 25px;
+  border: 1px solid black;
+  border-radius: 20px;
+  background-color: #FFFFFF;
+  font-size: 16px;
+  font-weight: 400;
+  margin-left: 1rem;
+  height: 44px;
+}
+.registerButton{
+  padding: 10px 25px;
+  width: 129px;
+  height: 44px;
+  background: #FF0000;
+  border: none;
+  border-radius: 20px;
+  color: #FFFFFF;
+  font-size: 16px;
+  margin-right: 1rem;
+  margin-left: 1rem;
+}
+
+@media (min-width: 992px) and (max-width: 1059px) {
+  .registerButton{
+    padding: 5px 10px;
+    width: 80px;
+    font-size: 13px;
+    margin-right: 0.5rem;
+    margin-left: 0.5rem;
+  }
+  .logInButton{
+    padding: 5px 10px;
+    width: 60px;
+    font-size: 13px;
+    margin-right: 0.5rem;
+    margin-left: 0.5rem;
+  }
+  .menuItems{
+    font-size: 13px;
+    margin-right: 0.5rem;
+    margin-left: 0.5rem;
+  }
+}
+@media (max-width: 991px) {
+  .navigationMenu{
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+  }
+  .divButtons{
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+  }
```

Comment:
What do you think about this, Ali?
`.divButtons {
  display: flex;
  flex-direction: row;
  gap: 10px;
  justify-content: center;
  align-items: center;
}`
![image](https://user-images.githubusercontent.com/36719203/176958024-74c9652a-3224-4df3-a0dd-3d6fd1633545.png)

Reply:
I didn't take notice of this. I will revise it.

Reply:
It's changed